### PR TITLE
Add Bogosort in Factor

### PR DESCRIPTION
**Pull request by nic-hartley**
added my username to CONTRIBUTORS.md too

### CODE REVIEW

**Comment by nic-hartley** [comment]
whoops, found a bug, fixing now

File: book.json [hunk 1]
```diff
@@ -139,7 +139,11 @@
         {
           "lang": "f90",
           "name": "Fortran90"
-        }
+        },
+        {
+          "lang": "factor",
```
Comment (jiegillet) [inline book.json]: #425 has this change already, you don't need it here.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -65,4 +65,6 @@ Trashtalk
 Cyrus Burt
 <br>
 Patrik Tesarik
-<br>
\ No newline at end of file
+<br>
+nic-hartley
```
Comment (jiegillet) [inline CONTRIBUTORS.md]: The formatting of that file has changed. Use the most recent version from master and add your name to it.

Comment (nic-hartley) [reply CONTRIBUTORS.md]: Fixed

File: book.json (hunk 1 above)
Comment (nic-hartley) [reply book.json]: #425 also hasn't been merged yet :)

**Pull request merged**